.github/workflows/ci.yml

```diff
@@ -104,6 +104,8 @@ jobs:
     with:
       project-name: Component[OpenTelemetry.Exporter.Geneva]
       code-cov-name: Exporter.Geneva
+      os-list: '[ "ubuntu-24.04" ]' # Note: This may be switched to latest once ubuntu-latest has a kernel version >= 6.8.0-1014-azure
```

Comment:
@xiang17 Just some context for you. From what I could tell, user_events were added to the kernel. But early versions seemed to have an issue where user space couldn't register the tracepoint as was intended. That was later fixed. So you need a kernel with user_events enabled AND the permissions fix for it all to work correctly.

How it should work is the reader process (think agent/collector) needs sudo/admin. But the writer process (app emitting telemetry) shouldn't need anything special.

Reply:
Thanks for the context. I simply added [your code](https://github.com/open-telemetry/opentelemetry-dotnet-contrib/pull/2219/files#diff-b803fcb7f17ed9235f1e5cb1fcd2f5d3b2838429d4368ae4c57ce4436577f03f) back which worked in your PR.

I removed your code because I [read it wrong](https://github.com/open-telemetry/opentelemetry-dotnet-contrib/pull/2316/commits/ca18ad34fe278e37f721799a8918ae79582e2478#r1876941768).

Reply:
However, simply adding back didn't work. The failure symptom is that the SDK is missing even though a correct SDK (9.0.101) is installed during setup dotnet step and dotnet restore & build steps were working:

```
  sudo -E dotnet test ./build/Projects/Component.proj --collect:"Code Coverage" --results-directory:TestResults --framework net9.0 --configuration Release --no-restore --no-build --logger:"console;verbosity=detailed" --filter "CategoryName=Geneva:user_events:metrics" -- RunConfiguration.DisableAppDomain=true && sudo chmod a+rw ./TestResults
  shell: /usr/bin/bash -e {0}
  env:
    BUILD_COMPONENT: OpenTelemetry.Exporter.Geneva
    DOTNET_ROOT: /usr/share/dotnet
The command could not be loaded, possibly because:
  * You intended to execute a .NET application:
      The application 'test' does not exist.
  * You intended to execute a .NET SDK command:
      A compatible .NET SDK was not found.

Requested SDK version: 9.0.100
global.json file: /home/runner/work/opentelemetry-dotnet-contrib/opentelemetry-dotnet-contrib/global.json

Installed SDKs:

Install the [9.0.100] .NET SDK or update [/home/runner/work/opentelemetry-dotnet-contrib/opentelemetry-dotnet-contrib/global.json] to match an installed SDK.

Learn about SDK resolution:
https://aka.ms/dotnet/sdk-not-found
8.0.110 [/usr/lib/dotnet/sdk]
Error: Process completed with exit code 145.
```

Reply:
I even changed the global.json version to 9.0.101 in case the installed 9.0.101 does not satisfy the `global.json`'s request for 9.0.100, however it still couldn't find the SDK:

```
  sudo -E dotnet test ./build/Projects/Component.proj --collect:"Code Coverage" --results-directory:TestResults --framework net9.0 --configuration Release --no-restore --no-build --logger:"console;verbosity=detailed" --filter "CategoryName=Geneva:user_events:metrics" -- RunConfiguration.DisableAppDomain=true && sudo chmod a+rw ./TestResults
  shell: /usr/bin/bash -e {0}
  env:
    BUILD_COMPONENT: OpenTelemetry.Exporter.Geneva
    DOTNET_ROOT: /usr/share/dotnet
The command could not be loaded, possibly because:
  * You intended to execute a .NET application:
      The application 'test' does not exist.
  * You intended to execute a .NET SDK command:
      A compatible .NET SDK was not found.

Requested SDK version: 9.0.101
global.json file: /home/runner/work/opentelemetry-dotnet-contrib/opentelemetry-dotnet-contrib/global.json

Installed SDKs:

8.0.110 [/usr/lib/dotnet/sdk]
Install the [9.0.101] .NET SDK or update [/home/runner/work/opentelemetry-dotnet-contrib/opentelemetry-dotnet-contrib/global.json] to match an installed SDK.

Learn about SDK resolution:
https://aka.ms/dotnet/sdk-not-found
Error: Process completed with exit code 145.
```